triggers.md

```diff
@@ -340,7 +353,17 @@ This has a few disadvantages:
 
 We worry that such a technique might encourage developers to insert hidden fake links into the DOM to trigger the heuristic.
 
+### Alternatives to `Speculation-Rules` header
+
+There is an existing header, “[Link][link-header],” which also allows for loading resources through headers. To use this, we would introduce a new [link type][link-type], "speculationrules." The header is semantically equivalent to the HTML `<link>` element, so using this header would mean we would have to support speculation rules loading with a `<link>` element as well. This seems sensible, especially given that web developers would already be familiar with loading declarative content with `<link>` elements (e.g. style sheets). However, this has a number of drawbacks. (1) For new kinds of resources to fetch, we want to require CORS. The `<link>` element has an unsuitable default in the form of the `crossorigin` attribute. (2) There are a number of other attributes which are undesirable or irrelevant for speculation rules (e.g. `type`). (3) The rel attribute can actually have multiple values, so this may lead to confusing combinations (e.g. rel="prefetch speculationrules"). (4) Since we already use `<script>` elements for inline speculation rules, it would be appropriate to support external speculation rules with `<script src=”...”>`, as noted [above](#external-speculation-rules-via-script-elements), but then we would have two ways of doing the same thing which could negatively impact developer experience.
```

Comment:
Could you reformat this ordered list so it gets the appropriate markdown styling, like:

```md
1. Foo
2. Bar
```

Reply:
Done.